OsmAnd/src/net/osmand/plus/backup/BackupInfo.java

```diff
@@ -89,11 +89,12 @@ private void createFilteredFilesToDownload(@NonNull OsmandApplication app) {
 	private void createFilteredFilesToUpload(@NonNull OsmandApplication app) {
 		List<LocalFile> files = new ArrayList<>();
 		BackupHelper helper = app.getBackupHelper();
+		boolean subscribedToPromo = InAppPurchaseHelper.isSubscribedToPromo(app);
```

Comment:
Strange, the promo subscription is checked in InAppPurchaseHelper.isOsmAndProAvailable(app) why is it not working? Probably there is an issue with a check for isAllowedInFreeVersion